src/pj_list.h

```diff
@@ -115,6 +119,7 @@ PROJ_HEAD(ortel, "Ortelius Oval")
 PROJ_HEAD(ortho, "Orthographic")
 PROJ_HEAD(pconic, "Perspective Conic")
 PROJ_HEAD(patterson, "Patterson Cylindrical")
+PROJ_HEAD(peirce_q, "Pierce Quincuncial")
```

Comment:
Pierce -> Peirce.

Charles Sanders Peirce: American semioticist, logician, geodesist and much more. Father of pragmaticism and abductive reasoning. Overall interesting figure (look him up one day you have too much time on your hands)